docker/sagemaker/Dockerfile.sm

```diff
@@ -18,7 +18,15 @@ FROM branch-${DEVICE} AS final
 LABEL maintainer="Amazon AI Graph ML team"
 
 # Install related Python packages
-RUN pip3 install ogb==1.3.6 scipy pyarrow boto3 scikit-learn transformers==4.28.1 \
+RUN pip3 install \
+        boto3 \
+        numba==0.58.1 \
+        numpy==1.26.1 \
```

Comment:
Does SageMaker partition rely on specify numpy and numba version?

Reply:
I've encountered this particular error when not pinning versions: https://github.com/numba/numba/issues/7339